docs/Tutorial_VALMOD.ipynb

```diff
@@ -0,0 +1,606 @@
+{
```

Comment:
<p>Should we really do this substitution now? Or can we do it at the end to match the paper?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
I tried to follow the paper as closely as possible. In fact, we do not even need to factor out `sigma_j,m/sigma_j,m+k`. We can start taking derivatives with respect to variables `mu_i,m+k` and `sigma_i,m+k`. The manipulation was performed by the authors and I basically tried to follow their steps.

Also, note that the factor `sigma_j,m/sigma_j,m+k` extracted by the authors will remain in eq(2). So, I thought it would be easier to follow the authors' step as they created such factor right from the beginning.... and focused on minimizing the (manipulated) term under the root sign.

If you want to avoid such substitution, then I suggest to avoid the factoring-out step as well, and simply treat  `mu_i,m+k` and `sigma_i,m+k` as variables right from the beginning.

Reply:
Agreed. Thinks look nicer as a result of the substitution